Amethyst/Managers/ScreenManager.swift

```diff
@@ -346,6 +382,16 @@ extension ScreenManager: Comparable {
 }
 
 extension WindowManager: ScreenManagerDelegate {
+    func applyWindowLimit(forScreenManager screenManager: ScreenManager<WindowManager<Application>>, minimizingIn range: (Int) -> Range<Int>) {
+        guard let screen = screenManager.screen else {return}
+        let windows =
+            screenManager.currentLayout is FloatingLayout
```

Comment:
I'm a little concerned about doing layout-specific logic outside of the layout itself. Especially with the introduction of custom layouts it could get unwieldy. Maybe the range should come from the layout itself? You can add some default behavior that can be overridden.

Reply:
Actually, on further consideration I think this is probably fine for now.